user-storage-simple/src/main/java/org/keycloak/quickstart/readonly/PropertyFileUserStorageProvider.java

```diff
@@ -117,7 +117,7 @@ public boolean isValid(RealmModel realm, UserModel user, CredentialInput input)
         UserCredentialModel cred = (UserCredentialModel)input;
         String password = properties.getProperty(user.getUsername());
         if (password == null) return false;
-        return password.equals(cred.getValue());
+        return password.equals(cred.getChallengeResponse());
```

Comment:
Remove this - we need to fix API issues on the server side

Reply:
I sent PR to fix this on server side. If you have a chance to review, it is here keycloak/keycloak#6795

Reply:
Resolving this comment, see https://github.com/keycloak/keycloak-quickstarts/pull/166#discussion_r380572963